dev-manuals/repositories/add-license-notices-at-pr.md

```diff
@@ -1,6 +1,6 @@
 # How to check, add and update license notices at every PR
```

Comment:
Also, what I noticed but forgot to criticize in the other PR: What about `keeping-licenses-up-to-date.md` as filename?
Because I don't think we can keep the current filename due to being unclear and not English.